src/i18n/ja/index.js

```diff
@@ -1,11 +1,156 @@
 import potato from './potato';
+import print from './print';
+import tester from './tester';
 export default {
   ja: {
     message: {
       ...potato,
+      help: {
+        label: 'ウィザード召喚'
+      },
+      print: { ...print },
+      tester: { ...tester },
       keyboard: {
-        label: 'きーぼーど'
-      }
+        label: 'keyboard'
+      },
+      layout: {
+        label: 'layout'
+      },
+      keymapName: {
+        label: 'Keymap Name',
+        placeholder: 'カスタムキーマップ名'
+      },
+      loadDefault: {
+        label: 'Load Default',
+        title: 'QMK Firmwareからデフォルトキーマップをロード'
+      },
+      compile: {
+        label: 'Compile',
+        title: 'キーマップのコンパイル'
+      },
+      downloadKeymap: {
+        title: 'keymap.cのみダウンロード',
+        label: 'keymap only'
+      },
+      downloadSource: {
+        title: 'ソースコードのダウンロード',
```

Comment:
```suggestion
        title: 'QMK全ソースコードのダウンロード',
```
To make the meaning clearer.
上との対比のため、`QMK全` を追加すると分かりやすいのではないでしょうか？

Reply:
Resolved in a6944e4